app/templates/components/modals/event-delete-modal.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="header">
+  {{t 'Are you sure you would like to delete this event?'}}
+  <div class="muted small text">
+    {{t 'Deleting the event will delete all the data associated with it.'}}
+  </div>
+</div>
+<div class="content">
+  <form class="ui form" autocomplete="off" {{action deleteEvent on='submit' preventDefault=true}}>
```

Comment:
Just putting 

`{{action (optional dummy) on='submit' preventDefault=true}}` should work